### ISSUE TITLE
How to reset the "Availability"?

### ISSUE THREAD

**Issue by smith111707** [pi-node/instructions#200]
Subj, please

**Comment by sepehrrostami** [comment]
Hi. It happened to me two times and that was because I think I installed the node again after I change my windows and tried a new VPN for Pi. But CT said that they have all the history of installation and the first installation too

**Comment by ihatejam** [comment]
I don't believe this can be set -- i'm pretty sure availability is a calculation -- based on your node installation's actual availability when "viewed" from outside of your network. Re-installing it will likely zero your availability, since it is seen as a new node setup (irrespective of whether you ran a node before or not).

**Comment by etyanxj** [comment]
Other
Pi Node version number: 0.4.5

Operating system: WINDOWS 10.0.19042 (Microsoft Windows 10 Home)

Using Docker Toolbox: No

Node installation date: 2021-03-22

Availability (up to 90 days): 8.99% from 2021-08-18 to 2021-11-16
